modules/minimint-wallet/src/config.rs

```diff
@@ -27,6 +29,8 @@ pub struct WalletClientConfig {
     pub peg_in_descriptor: PegInDescriptor,
     /// The bitcoin network the client will use
     pub network: Network,
+    /// The delay in blocks until a peg_in becomes available
```

Comment:
Maybe just change this comment to `Confirmations required for a peg in to be accepted by federation`?